css/style.css

```diff
@@ -0,0 +1,364 @@
+
+  font-family: 'Changa', sans-serif;
+  font-size: 30px;
+  font-weight: 700;
+  text-transform: uppercase;
+  line-height: 50px;
+  text-align: left;
+}
+
+h2{
+  display: inline-block;
+  padding: 20px;
+
+  font-family: 'Changa', sans-serif;
+  font-size: 20px;
+  font-weight: 700;
+  text-transform: uppercase;
+
+  border-bottom: 2px solid #d0cfd0;
+}
+
+header {
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
+  justify-content: space-around;
+  align-items: center;
+
+  height: 250px;
+    
+  color: #ffffff;
+  background-color: #607d8b;
+}
+
+.avatar{
+  margin-left: 30px;
+}
+
+.avatar img{
+  border-radius: 50%;
+  border:2px solid #bb946c;
+
+}
+
+.contacts{
+  margin-right: 20px;
+  text-align: left;
+}
+
+.contacts li{
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.contacts li::before{
+  content: "";
+  display: block;
+  width: 30px;
+  height: 30px;
+
+  position: absolute;
+  left: -40px;
+  top: -7px;
+
+  border: 1px solid #ffffff;
+  border-radius: 50%;
+
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 70%;
+}
+
+.contacts .number::before{
+  background-image: url("../img/contacts/tel.png");
+}
+
+.contacts .mail::before{
+  background-image: url("../img/contacts/email.png");
+}
+
+.contacts .skype::before{
+  background-image: url("../img/contacts/skype.png");
+}
+
+.contacts .location::before{
+  background-image: url("../img/contacts/location.png");
+}
+
+main{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-rows: 280px;
+  grid-column-gap: 10px;
+  grid-row-gap: 10px; 
+  padding: 10px;
+
+  background-color: #ffffff;
+}
+
+main section{
+  border: 1px solid #d0cfd0;
+}
+
+main p{
+  margin: 20px;
+}
+
+main ul{
+  margin: 20px;
+}
+
+.about{
+  grid-column-start: 1; 
+  grid-column-end: 2; 
+  grid-row-start: 1; 
+  grid-row-end: 2; 
+
+  text-align: justify;
+}
+
+.about p{
+  color: #6b6866;
+  line-height: 18px;
+}
+
+.education{
+  display: grid;
+
+  grid-row-start: 1; 
+  grid-row-end: 3; 
+}
+
+.styding ul{
+  margin-bottom:  50px;
+}
+
+.education li, .languages li{
+  margin-bottom: 10px;
+}
+
+.bold{
+  font-weight: 700;
+}
+
+.light{
+  font-size: 0.9em;
+  color: #607d8b;
+}
+
+.skills{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+
+  grid-column-start: 1; 
+  grid-column-end: span 2; 
+}
+
+.skills ul{
+  margin-left: 70px;
+}
+
+.skills li{
+  margin-top: 20px;
+  position: relative;
+}
+
+.skills li::before{
+  content: "";
+  display: block;
+  width: 30px;
+  height: 30px;
+
+  position: absolute;
+  left: -40px;
+  top: -7px;
+
+  border: 1px solid #bb946c;
+  border-radius: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 70%;
+}
+
+.skills li::after{
+  content: "";
+  display: block;
+  width: 150px;
+  height: 20px;
+
+  position: absolute;
+  top: -3px;
+  right: 70px;
+
+  border: 1px solid #bb946c;
+  border-radius: 10px;
+}
+
+.skills__hard li:nth-child(1)::before{
```

Comment:
it could be nice to use class here, as you did before (.contacts .number::before)